snapcraft/wrappers/lxd-stophook

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+# Use exec so that this script process is replaced.
+# This avoids polluting the process tree with this wrapper script.
+if [ "$1" = "callhook" ]; then
+    exec /snap/lxd/current/bin/lxd-user "$@"
+fi
+
+exec lxd "$@"
```

Comment:
@tomponline It just occurred to me that we may not want to fallback to invoking `lxd` (and relying on `$PATH`) for the non-callhook scenario.

If we ended up in that `lxd-stophook` script for a non-callhook, I think it'd be cleaner to `exit 1` with an error message explaining what's wrong.

Reply:
noted, thanks, will try this!